client/src/LoginForm.js

```diff
@@ -0,0 +1,149 @@
+
+const useStyles = makeStyles({
+  switch: {
+    position: "absolute",
+    top: '30px',
+    right: '30px',
+  },
+  forms: {
+    background: 'white',
+    border: 0,
+    borderRadius: 3,
+    color: 'white',
+    height: '100vh',
+    padding: '0 30px',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  button: {
+    maxWidth: '200px',
+    maxHeight: '70px',
+    minWidth: '200px',
+    minHeight: '70px',
+  },
+
+});
+
+
+const LoginForm = (props) => {
+  const classes = useStyles();
+  const history = useHistory();
+  const { user, login } = props;
+
+  const handleLogin = async (event) => {
+    event.preventDefault();
+    const username = event.target.username.value;
+    const password = event.target.password.value;
+
+    await login({ username, password });
+  };
+
+  if (user.id) {
+    return <Redirect to="/home" />;
+  }
+
+  return (
+    <div>
+        <Grid container className={classes.switch} direction="row" justify="flex-end" alignItems="center" spacing={2}>
+          <Grid item>
+            <Typography variant="h6" color="secondary">Don't have an account?</Typography>
+          </Grid>
+          <Grid item>
+            <Box boxShadow={3}>
+              <Button onClick={() => history.push("/register")} className={classes.button}>
+                  Create Account
+              </Button>
+            </Box>
+          </Grid>
+        </Grid>
+        
+        <Grid container direction="column" justify="center" alignItems="center" xm className={classes.forms}>
+            
+            <form onSubmit={handleLogin}>
+            <Grid
+              style = {{minWidth: "40vw"}} 
+              container
+              direction="column"
+              justify="center"
+              alignItems="left"
+              spacing={3}
+            >
+                <Grid item>
+                  <Typography variant="h4" color="textPrimary">
+                    Welcome back!
+                  </Typography>
+                </Grid>
+                <Grid item>
+                  <FormControl margin="normal" required fullWidth>
+                    <TextField
+                      aria-label="username"
+                      label="E-mail address"
+                      name="username"
+                      type="text"
+                    />
+                  </FormControl>
+                </Grid>
+                <Grid item>
+                  <FormControl margin="normal" required fullWidth>
+                    <TextField
+                      label="Password"
+                      aria-label="password"
+                      type="password"
+                      name="password"
+                      InputProps={{endAdornment:
+                      <Button>
+                        <Typography variant="h6" color="primary" disableFocusRipple disableRipple>
+                          Forgot?
+                        </Typography>
+                      </Button>}
```

Comment:
You don't have to include the "Forgot" button, but in the future, you should wrap this in an [`InputAdornment`](https://material-ui.com/components/text-fields/#input-adornments) component.